doc/sphinx/language/extensions/index.rst

```diff
@@ -4,7 +4,7 @@
 Language extensions
 ===================
 
-Elaboration extends the language accepted by the Rocq kernel to make it
+Syntax extensions and :term:`pretyping` extend the language accepted by the Rocq kernel to make it
```

Comment:
The problem was before this PR, but I find "extends the language accepted by the kernel" inaccurate.